src/lib.rs

```diff
@@ -364,8 +364,8 @@ fn from_public_key(source: &str) -> Result<(u8, [u8; 32])> {
     Ok((prefix, public_key))
 }
 
-/// Returns the type and the seed
-fn from_seed(source: &str) -> Result<(u8, [u8; 32])> {
+/// Attempts to decode the provided base32 encoded string into a valid prefix byte and the private key seed bytes
```

Comment:
Probably worth adding a note that this is for advanced usage (and that most people should use the normal `from_seed` method on the different key types) and is the same function used when turning a string into an nkey.

Reply:
Sounds good, I added a `NOTE: ...` comment below this line exactly to that effect.